avalon/pipeline.py

```diff
@@ -1383,13 +1386,10 @@ def format_template_with_optional_keys(data, template):
     work_file = template.format(**data)
 
     # Remove optional symbols
-    work_file = work_file.replace("<", "")
-    work_file = work_file.replace(">", "")
-
-    return work_file
+    return work_file.replace("<", "").replace(">", "").replace("..", ".")
```

Comment:
this is not adding anything functional. Please let's leave it in the syntax it was.

Reply:
added `.replace("..", ".")`

Reply:
yes, but also reformated the syntax. It's a matter of opinion, but I'd prefer leaving it as individual lines